src/main/java/org/opentripplanner/routing/impl/StreetVertexIndex.java

```diff
@@ -132,6 +139,10 @@ public VertexLinker getVertexLinker() {
     return vertexLinker;
   }
 
+  public TransitStopVertex findTransitStopVertices(FeedScopedId stopId) {
```

Comment:
The method should be probably in singular, i.e. `findTransitStopVertex`

Reply:
I am waiting for the naming conventions, all class/method names need to be looked at...